keyboards/s_ol/0xc_pad/readme.md

```diff
@@ -0,0 +1,24 @@
+# 0xC.pad
+
+![0xC.pad](https://i.imgur.com/YXMKy4u.png)
+
+A 12-key, RGB-backlit hexagonal macropad.
+
+* Keyboard Maintainer: [s-ol](https://github.com/s-ol)
+* Hardware Supported: "hex12pad r0", 0xC.pad r1
+* Hardware Availability: [IC form](https://s-ol.nu/0xC.pad)
+
+To flash the example keymap for this macropad
+(after setting up your build environment):
+​
+    make s_ol/0xc_pad:default:flash
```

Comment:
You have a couple of zero-width spaces in this file that break the intended rendering.

```suggestion
(after setting up your build environment):

    make s_ol/0xc_pad:default:flash
```

Reply:
Thanks, I'll investigate my editor settings to understand how this happened (and why I didn't notice)